rules/windows/lateral_movement_gpo_schtask_service_creation.toml

```
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2020/08/13"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/13"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects creation of new Group Policy based Scheduled Task or Service. Both methods are used for legit system
+administration but can be abused by an attacker with domain admin level to execute a malicious payload remotely on all
+or a subset of the domain joined machines. Verify this change context such as username, process and GPO XML content itself.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Creation of a new GPO Scheduled Task or Service"
```

```suggestion
name = "Creation or Modification of a new GPO Scheduled Task or Service"
```

Since it will be `event.type:(creation or change)` as the query is written